src/go/gateway/config/grpc.go

```diff
@@ -20,9 +21,12 @@ func init() {
 		// Interceptors can be the group name of UnaryClientInterceptorGroups in app/grpc/kernel.go.
 		"clients": map[string]any{
 			"user": map[string]any{
-				"host":         config.Env("GRPC_USER_HOST", ""),
-				"port":         config.Env("GRPC_USER_PORT", ""),
-				"handlers":     []gateway.Handler{protouser.RegisterUserServiceHandler},
+				"host": config.Env("GRPC_USER_HOST", ""),
+				"port": config.Env("GRPC_USER_PORT", ""),
```

Comment:
Use `GRPC_PACKAGE_HOST` and `GRPC_PACKAGE_PORT`.

Reply:
Yeah, most of these comment is already solved. I haven't pushed these changes to github yet.